app/server/models/transfer_usage.py

```diff
@@ -63,6 +63,7 @@ def find_or_create(cls, raw_name, default=False, **kwargs) -> "TransferUsage":
         if usage is None:
             usage = cls(name=name, default=default, **kwargs)
             db.session.add(usage)
+            db.session.flush()
```

Comment:
If we don't flush here, then we won't have an ID to tie the new transfer usage to the user